amgut/lib/data_access/test/test_ag_data_access.py

```diff
@@ -460,7 +464,9 @@ def test_getAvailableBarcodes(self):
 
         i = "d6b0f287-b9d9-40d4-82fd-a8fd3db6c476"
         res = self.ag_data.getAvailableBarcodes(i)
-        exp = ['000028434']
+        exp = [x['barcode'] for x in
```

Comment:
Does it change based on the order in which the tests are executed?

The problem that I'm seeing here is that at the end the function self.ag_data.getAvailableBarcodes is going to be similar to self.ag_data.ut_get_barcode_from_ag_login_id which at the end is not executing any unit testing correctly, but rather checking that the "same" function returns the same value twice